schemas/didDocument/didDocument.json

```diff
@@ -0,0 +1,33 @@
+{
+    "$id": "/did-core.didDocument",
+    "type": "object",
+    "properties": {
+      "@context": {
+        "title": "@context",
+        "description": "https://w3c.github.io/did-core-registry/#context",
+        "anyOf": [
+          { "type": "string" },
+          { "type": "array" },
+          { "type": "object" }
+        ],
+        "required": false
+      },
+      "id": {
+        "title": "DID",
+        "description": "https://w3c.github.io/did-core-registry/#id",
+        "type": "string",
+        "required": true,
```

Comment:
At this point, I think that `id` is the only thing that's required for a valid DID Document... we may want to ratchet that up and say that authentication is required as well, at a minimum... because DID Documents are useless w/o at least one verification method or controller? I expect some might disagree with that statement... like DID blockchain-based revocation lists... but I expect that those documents either have a controller or associated capabilityDelegation/Invocation verification methods. Thinking out loud more than anything... @dlongley, thoughts?

Reply:
I don't think we should require `authentication` as other verification relations might be used instead -- without enabling authentication. `id` is the only thing that's required. And while the expectation is that you might have at least one other verification method there, you may also have zero and perhaps have a `controller` instead.